tests/gamma/csm_observability_test.py

```diff
@@ -29,6 +31,51 @@
 _XdsTestClient = xds_k8s_testcase.XdsTestClient
 _Lang = skips.Lang
 
+# Testing consts
+_TEST_RUN_SECS = 90
+_REQUEST_PAYLOAD_SIZE = 271828
+_RESPONSE_PAYLOAD_SIZE = 314159
+_GRPC_METHOD_NAME = "grpc.testing.TestService/UnaryCall"
+_CSM_WORKLOAD_NAME_SERVER = "csm_workload_name_from_server"
+_CSM_WORKLOAD_NAME_CLIENT = "csm_workload_name_from_client"
+_CSM_CANONICAL_SERVICE_NAME_SERVER = "csm_canonical_service_name_from_server"
+_CSM_CANONICAL_SERVICE_NAME_CLIENT = "csm_canonical_service_name_from_client"
+_METRIC_CLIENT_ATTEMPT_SENT = "prometheus.googleapis.com/grpc_client_attempt_sent_total_compressed_message_size_bytes/histogram"
+_METRIC_CLIENT_ATTEMPT_RCVD = "prometheus.googleapis.com/grpc_client_attempt_rcvd_total_compressed_message_size_bytes/histogram"
+_METRIC_CLIENT_ATTEMPT_DURATION = (
+    "prometheus.googleapis.com/grpc_client_attempt_duration_seconds/histogram"
+)
+_METRIC_CLIENT_ATTEMPT_STARTED = (
+    "prometheus.googleapis.com/grpc_client_attempt_started_total/counter"
+)
+_METRIC_SERVER_CALL_RCVD = "prometheus.googleapis.com/grpc_server_call_rcvd_total_compressed_message_size_bytes/histogram"
+_METRIC_SERVER_CALL_SENT = "prometheus.googleapis.com/grpc_server_call_sent_total_compressed_message_size_bytes/histogram"
+_METRIC_SERVER_CALL_DURATION = (
+    "prometheus.googleapis.com/grpc_server_call_duration_seconds/histogram"
+)
+_METRIC_SERVER_CALL_STARTED = (
+    "prometheus.googleapis.com/grpc_server_call_started_total/counter"
+)
+_HISTOGRAM_CLIENT_METRICS = [
+    _METRIC_CLIENT_ATTEMPT_SENT,
+    _METRIC_CLIENT_ATTEMPT_RCVD,
+    _METRIC_CLIENT_ATTEMPT_DURATION,
+]
```

Comment:
here and below: immutable lists (especially if constants) should be converted to tuples
```suggestion
_HISTOGRAM_CLIENT_METRICS = (
    _METRIC_CLIENT_ATTEMPT_SENT,
    _METRIC_CLIENT_ATTEMPT_RCVD,
    _METRIC_CLIENT_ATTEMPT_DURATION,
)
```

Reply:
I kept these as arrays for now because there are some `+` operations just slightly below. Is it OK?

Reply:
Switched to tuple. The problem I had before probably had to do with the `b` in the example above only having 1 value, so that wasn't allowed:
```
>>> a = (1,2,3)
>>> b = (4)
>>> c = a + b
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: can only concatenate tuple (not "int") to tuple
```
Turns out i need to do this:
```
>>> b = (4,)
>>> c = a + b
>>> print(c)
(1, 2, 3, 4)
```

Reply:
Yea, it's a Python thing. Single-value tuples must be declared with a comma to distinguish from just in value in parens.